BuildScript.Nuke/ProjectMetadata.cs

```diff
@@ -1,14 +1,49 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Linq;
 using Nuke.Common.IO;
+using Nuke.Common.ProjectModel;
 
 #pragma warning disable CS8618
 
-internal class ProjectMetadata
+public class ProjectMetadata
 {
-  public AbsolutePath Path { get; init; }
+  public Project Project { get; init; }
+  public string Configuration { get; init; }
+  public AbsolutePath ProjectPath { get; init; }
   public string ToolsVersion { get; init; }
   public bool IsMultiTargetFramework { get; init; }
+  public IReadOnlyCollection<string> TargetFrameworks { get; set; }
```

Comment:
What about using `init` here?

Reply:
Wegen den init statt set in ProjectMetadata, das haben wir gemacht weil in CreateTestConfiguration teilen wir die ProjectMetadata auf für die Test Configurationen, dort clonen wir die ProjectMetadata und überschreiben die zwei Listen.
Sollen wir da set lassen oder sollen wir das in CreateTestConfiguration umschreiben.